xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h

```diff
@@ -487,6 +488,7 @@ extern "C" {
     int    iLifetime;                                     /*!< @brief (optional) life time in days of this recording */
     int    iGenreType;                                    /*!< @brief (optional) genre type */
     int    iGenreSubType;                                 /*!< @brief (optional) genre sub type */
+    const char *    strGenreDescription;                  /*!< @brief (optional) genre. Will be used only when iGenreType = EPG_GENRE_USE_STRING */
```

Comment:
we use character buffers, not pointers to addon internal structure. please change. buffer size could be PVR_ADDON_DESC_STRING_LENGTH.